File: app/src/main/java/com/hyapp/achat/model/LottieMessage.java
package com.hyapp.achat.model;

import com.aghajari.rlottie.AXrLottieDrawable;

public class LottieMessage extends ChatMessage {

    private AXrLottieDrawable drawable;

    public LottieMessage(byte transferType, long timeMillis, String uid, Contact sender, String receiverId, AXrLottieDrawable drawable) {
        super(TYPE_LOTTIE, transferType, timeMillis, uid, sender, receiverId);
        this.drawable = drawable;
    }

    public AXrLottieDrawable getDrawable() {
        return drawable;
    }

    public void setDrawable(AXrLottieDrawable drawable) {
        this.drawable = drawable;
    }
}

File: app/src/main/java/com/hyapp/achat/model/entity/LottieMessage.kt
package com.hyapp.achat.model.entity

import com.aghajari.rlottie.AXrLottieDrawable

class LottieMessage(
        uid: String = "",
        transfer: Byte = TRANSFER_RECEIVE,
        time: Long = 0,
        sender: Contact = Contact(),
        receiverUid: String = "",

        val drawable: AXrLottieDrawable? = null

) : ChatMessage(uid, TYPE_LOTTIE, transfer, time, sender, receiverUid)